docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform.md

```diff
@@ -72,7 +72,7 @@ Choose your repository, your branch, and keep "Autodeploy code changes" checked
 Here you will configure how DigitalOcean App Platform deploys your Strapi app. You can leave most things default. The only things you need to change are shown below:
 
 - **Environment Variables**: Add `DATABASE_URL`: `${db.DATABASE_URL}`
-- **Build Command**: `NODE_ENV=production npm run build`
+- **Build Command**: `DATABASE_URL=${DATABASE_URL} NODE_ENV=production npm run build`
```

Comment:
The database URL doesn't need to exist during the build process only the NODE_ENV and if you are an enterprise user then STRAPI_LICENSE (If you set a variable the URL key for the server.js that should also exist as well)